internal/generator/service.go

```diff
@@ -0,0 +1,185 @@
+package generator
```

Comment:
This thing is mostly copy/paste from the values handler. It does the wiring and detects if the the provided config version is a major tag range (e.g. "1.x.x") or a branch name so we don't have to repeat that logic in cmd/operator mode.